package.json

```diff
@@ -57,13 +57,12 @@
     "superagent": "8.1.2"
   },
   "devDependencies": {
-    "chai": "4.4.1",
+    "@types/jest": "29.5.12",
```

Comment:
Same here, we might not need types here

Reply:
This is to allow the editor to give better auto complete support.